appveyor.yml

```diff
@@ -105,5 +148,5 @@ test_script:
 #   - appveyor PushArtifact cbtmp.7z
 
 on_success:
-  - pip install codecov
+  - conda install -c conda-forge codecov -y
```

Comment:
Installing ``codecov`` from conda-forge to avoid any kind of problem regarding permissions when installing it using ``pip`` on appveyor

Reply:
@jjhelmus maybe `codecov` should be in `defaults` :wink:

Reply:
Building this out now